codecs/src/aper/mod.rs

```diff
@@ -221,6 +211,56 @@ impl AperCodecData {
     /// Dump current 'offset'.
     #[inline]
     pub fn dump(&self) {
-        log::trace!("AperCodecData: offset: {}", self.offset);
+        log::trace!("AperCodecData: offset: {}", self.decode_offset);
+    }
+
+    // Encoding functions.
+
+    /// Encode a bool.
+    fn encode_bool(&mut self, value: bool) -> Result<(), AperCodecError> {
+        self.bits.push(value);
+        Ok(())
+    }
+
+    /// Add bits to the encoding buffer.
+    fn append_bits(&mut self, bits: &BitSlice<Msb0, u8>) -> Result<(), AperCodecError> {
```

Comment:
I don't see a reason why this would fail, so may be there is no need to return a `Result`?

Reply:
Removed the result, and did the same to append_aligned() and encode_bool().